README.md

```diff
@@ -93,40 +93,53 @@ are marked with `gitlab: true` as in the configuration sample above. If you
 wish to disable gitlab authentication to any package config, just remove the
 element from the config.
 
-### Access
+### Permissions
 
-*access* is allowed depending on the following verdaccio `package` configuration
-directives:
+In order to define the way users can interact with packages, `verdaccio-gitlab`
+handles the following internal groups to determine permissions:
 
-- authenticated users are able to access all packages
-- unauthenticated users will be able to access packages marked with either
-  `$all` or `$anonymous` access levels at the package group definition
+- `$all` or `$anonymous`: Any user can perform the action on the packages. No
+condition at all.
+- `$authenticated`: Any successfully authenticated user can perform the action
+on the packages.
+- `$owned-group`: A user can perform the action on a package if
```

Comment:
We need to take care on the access level and I think there is no need for `$owned-group` as we have access level for this.

Reply:
See my answer above, I would need to alter the way the extension retrieve the user's data.